src/olcs/FeatureConverter.js

```diff
@@ -352,17 +356,19 @@ exports.prototype.createStackedGroundCorridors = function(layer, feature, width,
     width *= 2.14;
     const geometryOptions = {
       // always update Cesium externs before adding a property
-      positions,
       width,
       vertexFormat: Cesium.VertexFormat.POSITION_ONLY
     };
-    geometryInstances.push(new Cesium.GeometryInstance({
-      geometry: new Cesium.CorridorGeometry(geometryOptions),
-      attributes: {
-        color: Cesium.ColorGeometryInstanceAttribute.fromColor(color),
-        distanceDisplayCondition: new Cesium.DistanceDisplayConditionGeometryInstanceAttribute(previousDistance, distance - 1)
-      }
-    }));
+    for (const linePositions of positions) {
+      geometryOptions.positions = linePositions;
```

Comment:
It looks strange to change the `geometryOptions` object like this. You should move here the creation of the object so that a new object is created at each iteration of the loop.

Reply:
Interesting- I was specifically trying to avoid recreating the rest of the object each time. I assumed there would be a performance improvement that way.  I'll change it now for consistency though.

Reply:
The problem is that your previous code relied on an implementation detail in Cesium. If Cesium were to modify its behaviour our code would break.

Reply:
Isn’t that the case either way? Although I guess it’s easier to find the spot to change things if the object is made in one spot.